### PR TITLE
Add standard side bar urls in QFileDialog, fix issue #135

### PR DESCRIPTION
**Pull request by Hombre57**

### CODE REVIEW

File: src/FileSystem.cpp [hunk 1]
```diff
@@ -0,0 +1,70 @@
+/*
+ *  HDRMerge - HDR exposure merging software.
+ *  Copyright 2012 Javier Celaya
```
Comment (Beep6581) [inline src/FileSystem.cpp]: @Hombre57 you are the author, so it should be "Copyright 2018 <Your Name> <contactinfo>"

https://www.gnu.org/licenses/gpl-howto.html
> This involves adding two elements to each source file of your program: a copyright notice (such as “Copyright 1999 Terry Jones”), and a statement of copying permission, saying that the program is distributed under the terms of the GNU General Public License (or the Lesser GPL, or the Affero GPL).

http://gplv3.fsf.org/wiki/index.php/Howto

Comment (Beep6581) [inline src/FileSystem.cpp]: @Hombre57 you authored the file, so you have the copyright and no one else. You can licence it under the GPL, but you're still the copyright holder, so this line should be:
`Copyright 2018 <your name> <contact details>`

> This involves adding two elements to each source file of your program: a copyright notice (such as “Copyright 1999 Terry Jones”), and a statement of copying permission, saying that the program is distributed under the terms of the GNU General Public License (or the Lesser GPL, or the Affero GPL).

https://www.gnu.org/licenses/gpl-howto.en.html
http://gplv3.fsf.org/wiki/index.php/Howto

File: src/LoadOptionsDialog.cpp [hunk 2]
```diff
@@ -138,14 +139,22 @@ void LoadOptionsDialog::addFiles() {
...
-        lastDirSetting.isNull() ? QDir::currentPath() : QDir(lastDirSetting.toString()).absolutePath(),
-        filter, Q_NULLPTR, Q_NULLPTR);
-    if (!files.empty()) {
-        QString lastDir = QFileInfo(files.front()).absolutePath();
-        settings.setValue("lastOpenDirectory", lastDir);
-        for (auto & i : files) {
-            new FileItem(i, fileList);
+
+
+    QFileDialog loadDialog(this, tr("Open raw images"), lastDirSetting.isNull() ? QDir::currentPath() : QDir(lastDirSetting.toString()).absolutePath(), tr("Digital Negatives (*.dng)"));
```
Comment (Beep6581) [inline src/LoadOptionsDialog.cpp]: This line chaned HDRMerge into DNG-onlyMerge ;)
It needs the `filter` from above to allow opening raw files other than DNG.

Comment (Hombre57) [reply src/LoadOptionsDialog.cpp]: Damn, copypasta error :disappointed: Will be fixed in next commit.

**Comment by Beep6581** [comment]
@Hombre57 are there any modifications pending to this PR, or should we merge?

**Comment by Hombre57** [comment]
@Beep6581 I tried to find a filtering logic to keep only useful mounts, w/o success. I'd commit this PR and let someone find a correct way to filter the list. Maybe looking at Gtk's code can hep (though it can be odd too), but I won't have time to do it.

**Pull request merged**